src/content/configuration/stats.md

```diff
@@ -44,176 +44,686 @@ module.exports = {
 
 For more granular control, it is possible to specify exactly what information you want. Please note that all of the options in this object are optional.
 
-<!-- eslint-skip -->
+### `stats.all`
+
+A fallback value for stats options when an option is not defined. It has precedence over local webpack defaults.
+
+```javascript
+module.exports = {
+  //...
+  stats: {
+    all: undefined
```

Comment:
https://github.com/webpack/webpack/blob/master/lib/Stats.js#L112
this is so confusing, i've resued the previous information but would love to improve.